receiver/googlecloudspannerreceiver/metricsreader.go

```diff
@@ -25,6 +25,12 @@ import (
 	"google.golang.org/api/iterator"
 )
 
+const (
+	projectIdLabelName = "project_id"
```

Comment:
Does it make sense to add some prefix to labels, like "gcp", "sp" or "gcp_sp" or msth like that to denote that labels are domain specific?

Reply:
We agreed that label names will be exactly the same as in Cloud Monitoring, and currently they have the same names as in CM.